app/views/activities/index.html.haml

```diff
@@ -0,0 +1,25 @@
+.container
+  .row
+    .col-md-8
+      %h3.align-center.blue View Previous Logs
+      %form.form-inline
+        #date-range.input-group
+          %input.form-control.date-picker{:type => "text", :name => "start"} 
```

Comment:
Ok great i was looking at those and wondering /thinking the same thing = if that is where my data was coming from and if that is what i could pull from